rust/sbp/src/messages/acquisition.rs

```diff
@@ -0,0 +1,397 @@
+// Copyright (C) 2015-2018 Swift Navigation Inc.
+// Contact: Swift Navigation <dev@swiftnav.com>
+//
+// This source is subject to the license found in the file 'LICENSE' which must
+// be be distributed together with this source. All other rights reserved.
+//
+// THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+// EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+// WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+//****************************************************************************
+// Automatically generated from yaml/swiftnav/sbp/acquisition.yaml
+// with generate.py. Please do not hand edit!
+//****************************************************************************/
+// Satellite acquisition messages from the device.
+extern crate byteorder;
+#[allow(unused_imports)]
+use self::byteorder::{LittleEndian, ReadBytesExt};
+use super::gnss::*;
+
+// Satellite acquisition result
+//
+// This message describes the results from an attempted GPS signal
+// acquisition search for a satellite PRN over a code phase/carrier
+// frequency range. It contains the parameters of the point in the
+// acquisition search space with the best carrier-to-noise (CN/0)
+// ratio.
+//
+#[derive(Debug)]
+#[allow(non_snake_case)]
+pub struct MsgAcqResult {
+    pub sender_id: Option<u16>,
+    pub cn0: f32,
+    // ^ CN/0 of best point
```

Comment:
Put these before the field with `///` (for the same purpose as above).

Reply:
Done. This was caught by the cargo build when I turned all of the comments into doc comments!